README.md

```diff
@@ -1,19 +1,22 @@
-# [Tetragram] - [Task Name]
-[Link](url) to task
+# [lecr] - [JaniClean: Automation for Employee's Birthday Template]
+[Link](https://www.odoo.com/web#id=3364114&menu_id=4720&cids=3&action=4665&active_id=3364090&model=project.task&view_type=form) to task
 
 ## Steps to complete the dev
-- [X] Step 1
-- [ ] Step 2
+- [X] Install the HR module and create dummy employees with work emails and birthdays
```

Comment:
module name currently is birthday_automation_template but  should be <extention_modulename><short_feature_name>

Eg: janiclean_email_automation, or something down that line